magic-mock/public/index.html

```diff
@@ -7,12 +7,12 @@
         <h1>Welcome to Client Portal Version 999.99</h1>
         <a
             href="http://localhost:3001/api/auth/magic-callback?userID=userID&token=token"
-            target="_blank"
+            target="_self"
```

Comment:
no need to have the target, the default behavior is to use the same browser window

Reply:
false

Reply:
Also it's for reusing tab, not just browser window

Reply:
just to keep track of our conversation in slack. https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/target